lighthouse-core/report/html/report-styles.css

```diff
@@ -15,41 +15,58 @@
  * limitations under the License.
  */
 
+/*
+  Naming convention:
+
+  If a variable is used for a specific component: --{component}-{property name}-{modifier}
+
+  Both {component} and {property name} should be kebab-case. If the target is the entire page,
+  use 'report' for the component. The property name should be not be abbreviated. Use the
+  property name the variable is intended for - if it's used for multiple, a common descriptor
+  is fine (ex: 'size' for a variable applied to 'width' and 'height'). If a variable is shared
+  across multiple components, either create more variables or just drop the "{component}-"
+  part of the name. Append any modifiers at the end (ex: 'big', 'dark').
+
+  For colors: --color-{hue}-{intensity}
+
+  {intensity} is the Material Design tag - 700, A700, etc. Omit "-{intensity}" if there is only one
+  color of that hue.
+*/
 .lh-vars {
   --text-font-family: Roboto, Helvetica, Arial, sans-serif;
   --monospace-font-family: 'Roboto Mono', 'Menlo', 'dejavu sans mono', 'Consolas', 'Lucida Console', monospace;
   --body-background-color: #fff;
   --body-text-color: var(--color-black-900);
   --body-font-size: 16px;
   --body-line-height: 24px;
-  --subheader-color: hsl(206, 6%, 25%);
-  --header-font-size: 20px;
+  --snippet-color: hsl(206, 6%, 25%);
+  --category-header-font-size: 20px;
   --header-line-height: 24px;
-  --caption-line-height: 16px;
+  --audit-explanation-line-height: 16px;
   --default-padding: 12px;
-  --section-padding: 16px;
-  --section-indent: 16px;
-  --audit-item-gap: 5px;
-  --text-indent: 8px;
+  --footer-padding-vertical: 16px;
+  --audit-margin-horizontal: 5px;
+  --stackpack-padding-horizontal: 10px;
   --secondary-text-color: var(--color-black-800);
   --informative-color: var(--color-blue-900);
-  --medium-75-gray: #757575;
-  --medium-50-gray: hsl(210, 17%, 98%);
+  --color-gray-600: #757575;
+  --color-gray-50: #FAFAFA;
+  --snippet-background-color: var(--color-gray-50);
```

Comment:
There's actually a few of the color component variables defined in this manner already:

```
--report-text-color: var(--color-black-900);
--informative-color: var(--color-blue-900);
--snippet-background-color: var(--color-gray-50);
--tools-icon-color: var(--color-gray-600);
--env-item-background-color: var(--color-black-100);
--plugin-badge-background-color: var(--color-white);
--topbar-background-color: var(--color-black-100);
--metrics-toggle-background-color: var(--color-black-200);
```

which is kinda nice because inverting the `--color-` variables is all that needs to be done to do dark theming.

Reply:
or do you mean removing the variable `--color-gray-50` and inlining the value here